openedx_events/learning/signals.py

```diff
@@ -196,3 +197,25 @@
         "notification_data": UserNotificationData,
     }
 )
+
+# .. event_type: org.openedx.learning.user.add.course.staff.role.v1
+# .. event_name: ADD_COURSE_STAFF_ROLE
+# .. event_description: Assigns the "Course Staff" role to a user.
```

Comment:
I think this description is off because this event doesn't cause a course staff assignment, it's fired as a result of one. Might also want to tweak the event_name to be more clear on that as well.

Reply:
can you update the descriptions too?